canonical/greenhouse-automations
# Updated help descriptions of the commands

# Done
- Commands, arguments and options descriptions are updated.
- Added "Summarised Description" to every command and options to the [spec](https://docs.google.com/document/d/1zXq6ZtXi6UCUVOyF4_Fux13nDjePZWv-_JeEo0vytT0/edit#).

# QA
- Checkout the feature branch
- Install dependencies with `yarn`
- Run `ts-node ./src/index.ts --help` 
- Verify the program description is correct.
- Verify the usage is correct.
- Verify the "add-posts" command's description is correct.
- Verify the "delete-posts" command's description is correct.
- Run `ts-node ./src/index.ts add-posts --help`
- Verify the command's description is correct
- Verify the argument and options descriptions are correct.
- Verify the usage is correct. 
- Run `ts-node ./src/index.ts delete-posts --help`
- Verify the command's description is correct
- Verify the argument and options descriptions are correct.
- Verify the usage is correct.

Fixes #48
